File: 1833-find-the-highest-altitude/find-the-highest-altitude.cpp
class Solution {
public:
    int largestAltitude(vector<int>& gain) {
         vector<int> result(1,0);
        int ans=0;
        for(int i=0;i<gain.size();i++){
            
            ans=ans+gain.at(i);
            result.push_back(ans);

        }
        ans=0;
        for(int i=0;i<result.size();i++){
            
            if(ans<result.at(i)){
                ans=result.at(i);
            }

        }

        return ans;
    }
    
};

File: 1833-find-the-highest-altitude/find-the-highest-altitude.js
/**
 * @param {number[]} gain
 * @return {number}
 */
var largestAltitude = function(gain) {
    let total=0;
    let max=0;
    for(let i=0;i<gain.length;i++){
        total+=gain[i];
        max=Math.max(max,total);
    }
    return max;
    
};